packages/pebble-web/src/components/TimePicker.tsx

```diff
@@ -17,6 +17,9 @@ import { colors } from "pebble-shared";
 const HOURS = /*#__PURE__*/ [...Array(12)].map((_, i) =>
   ("00" + (i + 1).toString(10)).slice(-2)
 );
+const TWENTY_FOUR_HOUR = /*#__PURE__*/ [...Array(24)].map((_, i) =>
```

Comment:
```suggestion
const TWENTY_FOUR_HOURS = /*#__PURE__*/ [...Array(24)].map((_, i) =>
```

also change in call site